api/src/resources/comment.rs

```diff
@@ -189,6 +236,36 @@ pub struct SyncCommentsResponse {
     pub unchanged: usize,
 }
 
+impl ReducableResponse for SyncCommentsResponse {
+    fn empty() -> Self {
+        Self {
+            new: 0,
+            updated: 0,
+            unchanged: 0,
+        }
+    }
+
+    fn merge(a: Self, b: Self) -> Self {
+        let Self {
+            new: a_new,
+            updated: a_updated,
+            unchanged: a_unchanged,
+        } = a;
+
+        let Self {
+            new: b_new,
+            updated: b_updated,
+            unchanged: b_unchanged,
+        } = b;
+
+        Self {
+            new: a_new + b_new,
+            updated: a_updated + b_updated,
+            unchanged: a_unchanged + b_unchanged,
```

Comment:
I added the let binding so that we would get a compiler error if new fields were added (to remind us to make any necessary changes here)

Reply:
Rather than do that, you can just put a single statement like this at the top:
```rust
// Ensure new fields are added here:
let Self { new: _, updated: _, unchanged: _ } = a;
```

Reply:
Thanks - just pushed a slightly different implementation